j2/vars/custom-sharepoint-rules.yaml

```diff
@@ -49,7 +49,7 @@ rules:
 
       - label: Object_Module
         props:
-          Name: "${field_Tabela?.trim()?.toLowerCase()}"
+          Name: "${field_Tabela?.trim()?.toUpperCase()}"

```

Comment:
Please revert this back; this will break Sunny's environment

Reply:
rolled them back